python/huggingfaceserver/README.md

```diff
@@ -111,7 +111,7 @@ spec:
           cpu: 100m
           memory: 2Gi
 ```
-3. Serve the llama2 model using huggingface vllm runtime. For the llama2 model, vllm is the default model. If available for a model, vllm is set as the default runtime. Note - Model need to be supported by vllm otherwise KServe python runtime will be used as a failsafe.
+3. Serve the llama2 model using huggingface vllm runtime. For the llama2 model, vllm is supported and used as the default model. If available for a model, vllm is set as the default runtime. Note - Model need to be supported by vllm otherwise KServe python runtime will be used as a failsafe.
```

Comment:
```suggestion
3. Serve the llama2 model using huggingface vLLM runtime. For the llama2 model, vllm is supported and used as the default model. If available for a model, vLLM is set as the default runtime. Note - Model needs to be backed by vLLM otherwise KServe python runtime will be used as a failsafe.
```

Reply:
Can we use "backend" instead of "model" when referring to vLLM?